staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1969,3 +1976,19 @@ enum CgroupDriver {
     CGROUPFS = 1;
 }
 
+message UpdatePodSandboxResourcesRequest {
+    // ID of the PodSandbox to update.
+    string pod_sandbox_id = 1;
+
+    // Optional overhead represents the overheads associated with this sandbox
+    LinuxContainerResources overhead = 2;
+    // Optional resources represents the sum of container resources for this sandbox
+    LinuxContainerResources resources = 3;
+
```

Comment:
I've not had the chance to keep up with the motivation behind this change. But curious if we need to support Windwos as well here. Please see: https://github.com/kubernetes/kubernetes/blob/d67e6545b159658d5500f773595cc7a6b62e94ba/staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto#L1358-L1369

/cc @marosset

Reply:
We don't currently have plans to support resource update on windows. We can always add it if that changes.